packages/cms-lib/lib/config.js

```diff
@@ -417,11 +418,19 @@ const getEnv = nameOrId => {
   return env;
 };
 
-const getAccountConfig = accountId =>
-  getConfigAccounts(getAndLoadConfigIfNeeded()).find(
+const getAccountConfig = accountId => {
+  const account = getConfigAccounts(getAndLoadConfigIfNeeded()).find(
     account => account.accountId === accountId || account.portalId === accountId
   );
 
+  if (account && account.portalId) {
+    account.accountId = account.accountId || account.portalId;
+    delete account.portalId;
+  }
```

Comment:
Hmm...I was thinking that we would only do this when a user is going through a flow that is explicitly a write operation. If I understand how this is working, it will rewrite the config when going through the write process when exchanging an access key for an access token which happens transparently in the background when the access token is about to expire.

Reply:
Yeah I was thinking about this too.  I moved the delete to the updateAccountConfig function

Reply:
Can we move all side effects to the update? I think it makes more sense to keep this function pure.

Reply:
My concern of moving the `account.accountId = account.accountId || account.portalId` line would be that a function calls into `getAccountConfig`, making the assumption that `accountId` will be present, when in reality the config file hadn't been updated yet